deployments/server_alpha_db.sql

```diff
@@ -0,0 +1,187 @@
+-- Database generated with pgModeler (PostgreSQL Database Modeler).
+-- pgModeler version: 1.1.0-beta1
+-- PostgreSQL version: 16.0
+-- Project Site: pgmodeler.io
+-- Model Author: ---
+-- -- object: alpha | type: ROLE --
+-- -- DROP ROLE IF EXISTS alpha;
+-- CREATE ROLE alpha WITH 
+-- 	CREATEDB
+-- 	CREATEROLE
+-- 	LOGIN;
+-- -- ddl-end --
+-- 
+
+-- Database creation must be performed outside a multi lined SQL file. 
+-- These commands were put in this file only as a convenience.
+-- 
+-- -- object: server_alpha_db | type: DATABASE --
+-- -- DROP DATABASE IF EXISTS server_alpha_db;
+-- CREATE DATABASE server_alpha_db
+-- 	ENCODING = 'UTF8';
+-- -- ddl-end --
+-- 
+
+-- object: alpha_schema | type: SCHEMA --
+-- DROP SCHEMA IF EXISTS alpha_schema CASCADE;
+CREATE SCHEMA alpha_schema;
+-- ddl-end --
+ALTER SCHEMA alpha_schema OWNER TO alpha;
+-- ddl-end --
+
+SET search_path TO pg_catalog,public,alpha_schema;
+-- ddl-end --
+
+-- object: alpha_schema.users | type: TABLE --
+-- DROP TABLE IF EXISTS alpha_schema.users CASCADE;
+CREATE TABLE alpha_schema.users (
+	user_id uuid NOT NULL,
+	username varchar(25) NOT NULL,
+	nickname varchar(20) NOT NULL,
+	email varchar(128) NOT NULL,
+	password varchar(32) NOT NULL,
+	created_at date NOT NULL,
+	activated_at date,
+	expires_at date,
```

Comment:
Wir benutzen kein Date, das ist zu ungenau, da es nur den Tag datiert. Um ganz genau zu sein wie abgesprochen in der Vorlesung, brauchen wir timestamptz, also nen Timestamp mit Timezone. Unten im Schema dann genauso